src/routes/[...404].tsx

```diff
@@ -0,0 +1,8 @@
+import { useLocation } from "@solidjs/router";
+import { getLocaleFromPathname, missingTranslationMessage } from "~/i18n";
+
+export default function NoEntry() {
+	const { pathname } = useLocation();
+
+	return <p>{missingTranslationMessage(getLocaleFromPathname(pathname))}</p>;
```

Comment:
Considering my message on the `i18n-anchor.tsx` file. Should we add a non i18-anchor here to route back to the default language in case someone hits this?